config/mapping.json

```diff
@@ -1,30 +0,0 @@
-[
-	{
-		"regex":"gutenberg/packages/block-library/src/image/edit.native.js",
-		"testFile": ["gutenberg/image-upload.md", "gutenberg/image-caption.md"]
```

Comment:
The test files referenced in this file actually point to `test-suites` folder ([reference](https://github.com/wordpress-mobile/test-case-reminder-bot/blob/ca5a7e299e15d208926237215c3a0c6ab2b0b305/application_helper.rb#L37)). In this case, the test files are pointing to:
- `test-suites/gutenberg/image-upload.md`
- `test-suites/gutenberg/image-caption.md`

This is the main reason why I removed all the test suite files from `test-suites`.